scripts/create.sh

```diff
@@ -3,52 +3,47 @@
 
 echo "-> [create.sh]"
 
-# Make sure this script only repply on an Acorn creation event
+# Make sure this script only replies on an Acorn creation event
 if [ "${ACORN_EVENT}" != "create" ]; then
    echo "ACORN_EVENT must be [create], currently is [${ACORN_EVENT}]"
    exit 0
 fi
 
-# Check if cluster with that name already exit
-atlas cluster get ${CLUSTER_NAME} 2>/dev/null
-if [ $? -eq 0 ]; then
-  echo "-> cluster ${CLUSTER_NAME} already exists"
-
-  DB_ADDRESS=$(atlas cluster describe ${CLUSTER_NAME} -o json | jq -r .connectionStrings.standardSrv)
-  DB_PROTO=$(echo $DB_ADDRESS | cut -d':' -f1)
-  DB_HOST=$(echo $DB_ADDRESS | cut -d'/' -f3)
-  echo "DB_ADDRESS: [${DB_ADDRESS}] / DB_PROTO:[${DB_PROTO}] / DB_HOST:[${DB_HOST}]"
+# Check if project with that name does not already exit
+res=$(atlas project list -o json | jq -r --arg project_name "$PROJECT_NAME" '.results[] | select(.name == $project_name)')
+if [ "$res" != "" ]; then
+  echo "-> project ${PROJECT_NAME} already exists"
```

Comment:
Should we check that the instance type is M0 and the project exists? Since there can be only 1. I assume multiple other instances can be created.

Reply:
Also, there should be a helper function that writes the error cause to `/dev/termination-log` that will bubble up to the user in the UI.

Reply:
As a project will be created each time, I guess there is no need to check its existence, you'r right.
No problem, I'll make sure to use /dev/termination-log